### PR TITLE
doc: Update frdm_k64f doc about modbus fixtures

### PR DESCRIPTION
**Pull request by shaoanxli**
Add the modbus fixtures to frdm_k64f doc.

Signed-off-by: Shaoan Li <shaoanx.li@intel.com>

### CODE REVIEW

**Review by nashif — COMMENTED** [review]
what is the reason we are adding this specificially for this board? This applies to all boards and there is nothing special about frdm board when it comes to twister.

File: boards/arm/frdm_k64f/doc/index.rst [hunk 1]
```diff
@@ -321,6 +321,15 @@ should see the following message in the terminal:
    ***** Booting Zephyr OS v1.14.0-rc1 *****
    Hello World! frdm_k64f
 
+**note**
```
Comment (nashif) [inline boards/arm/frdm_k64f/doc/index.rst:324]: This should be documented in the test, not in the board documentation.

**Review by nashif — CHANGES_REQUESTED** [review]

File: boards/arm/frdm_k64f/doc/index.rst (hunk 1 above)
Comment (shaoanxli) [reply boards/arm/frdm_k64f/doc/index.rst]: Have updated to the test, can help check whether it is correct？

File: tests/subsys/modbus/src/main.c [hunk 2]
```diff
@@ -6,6 +6,14 @@
...
 #include "test_modbus.h"
 
+/*
+ * If need to test this test suite,
+ * should add the following fixtures to map.yml:
+ *
+ *   fixtures:
+ *   - uart_loopback
+ */
+
```
Comment (jfischer-no) [inline tests/subsys/modbus/src/main.c:16]: There is already comment about test configuration in https://github.com/zephyrproject-rtos/zephyr/blob/main/tests/subsys/modbus/src/test_modbus.h#L22, I would rather insert there: 
```diff
diff --git a/tests/subsys/modbus/src/test_modbus.h b/tests/subsys/modbus/src/test_modbus.h
index 0f38d7d908d..ce35d9dce3f 100644
--- a/tests/subsys/modbus/src/test_modbus.h
+++ b/tests/subsys/modbus/src/test_modbus.h
@@ -19,6 +19,9 @@
 #define MB_TEST_FP_OFFSET      5000
 
 /*
+ * This test performed on hardware requires two UART controllers
+ * on the board with RX/TX lines connected crosswise.
+ *
  * Integration platform for this test is FRDM-K64F.
  * The board must be prepared accordingly:
  * UART3(PTC16)-RX <-> UART2(PTD3)-TX pins and
```

**Review by jfischer-no — CHANGES_REQUESTED** [review]

File: tests/subsys/modbus/src/main.c (hunk 2 above)
Comment (jfischer-no) [reply tests/subsys/modbus/src/main.c]: unresolved, :arrow_up: comment like this or NACK

**Comment by dleach02** [comment]
@jfischer-no @nashif, author has provided updates to your comments. Please determine if they address your blocks.

**Review by jfischer-no — DISMISSED** [review]

**Comment by shaoanxli** [comment]
Is there anything else that needs to be changed on this pr?

**Review by jfischer-no — DISMISSED** [review]

**Review by jfischer-no — DISMISSED** [review]

**Comment by mengxianglinx** [comment]
Can anybody review this patch?

**Review by mmahadevan108 — APPROVED** [review]

**Review by mmahadevan108 — CHANGES_REQUESTED** [review]
Please update the commit message as the change is not K64F specific.

**Review by mmahadevan108 — APPROVED** [review]

**Comment by mmahadevan108** [comment]
@nashif Can you help re-review.

**Review by jfischer-no — COMMENTED** [review]
You are not updating documentation as such, rather the commit should be like:
```

tests: modbus: update comment about test fixture

Add more information about test fixture requirements.
```

**Pull request closed without merging**